src/components/ProfilePage.jsx

```diff
@@ -67,17 +67,30 @@ export class ProfilePage extends React.Component {
     this.props.saveProfile(formId);
   }
 
-  handleChange(formId, name, value) {
-    if (name === 'visibility') {
-      this.props.updateVisibilityDraft(formId, value);
-    } else {
-      this.props.updateAccountDraft(formId, value);
-    }
+  handleChange(name, value) {
+    this.props.updateDraft(name, value);
   }
 
   render() {
     const {
-      profileImage, username, dateJoined, errors,
+      profileImage,
+      username,
+      dateJoined,
+      errors,
+      name,
+      visibilityName,
+      country,
+      visibilityCountry,
+      education,
+      visibilityEducation,
+      socialLinks,
+      draftSocialLinksByPlatform,
+      visibilitySocialLinks,
+      languageProficiencies,
+      visibilityLanguageProficiencies,
+      visibilityCourseCertificates,
+      bio,
+      visibilityBio,
```

Comment:
We pass all the data down to the child forms now, rather than having them select it themselves.  Better symmetry.